src/gui/Src/Gui/HandlesView.cpp

```diff
@@ -42,12 +42,13 @@ HandlesView::HandlesView(QWidget* parent) : QWidget(parent)
     mWindowsTable->addColumnAt(8 + 8 * charWidth, tr("Handle"), true, "", StdTable::SortBy::AsHex);
     mWindowsTable->addColumnAt(8 + 120 * charWidth, tr("Title"), true);
     mWindowsTable->addColumnAt(8 + 40 * charWidth, tr("Class"), true);
-    mWindowsTable->addColumnAt(8 + 8 * charWidth, tr("Thread"), true, "", StdTable::SortBy::AsHex);
+    mWindowsTable->addColumnAt(8 + 16 * charWidth, tr("Thread ID"), true, "", StdTable::SortBy::AsHex);
+    mWindowsTable->addColumnAt(8 + 32 * charWidth, tr("Thread"), true);

```

Comment:
Adding a new column would break existing user's UI settings (since it restores the size/order of the wrong column). In this case I don't think it's necessary either because it just clutters the view.

Reply:
Column removed.